tests/integration/test_s3_plain_rewritable/test.py

```diff
@@ -75,9 +77,10 @@ def create_insert(node, insert_values):
     assert len(cluster.instances) == NUM_WORKERS
     for i in range(NUM_WORKERS):
         node = cluster.instances[f"node{i + 1}"]
-        t = threading.Thread(target=create_insert, args=(node, insert_values_arr[i]))
-        threads.append(t)
-        t.start()
+        for table_name, values in [("test", insert_values_arr[i]), ("test_dst", "")]:
+            t = threading.Thread(target=create_insert, args=(node, table_name, values))
+            threads.append(t)
+            t.start()
```

Comment:
Looks like the test became flaky - https://play.clickhouse.com/play?user=play#U0VMRUNUIGNoZWNrX3N0YXJ0X3RpbWUsIGNoZWNrX25hbWUsIHRlc3RfbmFtZSwgcmVwb3J0X3VybApGUk9NIGNoZWNrcwpXSEVSRSBjaGVja19uYW1lIExJS0UgJ0ludGVncmF0aW9uJScKICAgIEFORCBjaGVja19zdGFydF90aW1lID49IG5vdygpIC0gSU5URVJWQUwgMTQgREFZCiAgICBBTkQgKHN0YXJ0c1dpdGgoaGVhZF9yZXBvLCAnQ2xpY2tIb3VzZS8nKSkKICAgIEFORCB0ZXN0X3N0YXR1cyAhPSAnU0tJUFBFRCcKICAgIEFORCAodGVzdF9zdGF0dXMgTElLRSAnRiUnIE9SIHRlc3Rfc3RhdHVzIExJS0UgJ0UlJykgCiAgICBBTkQgY2hlY2tfc3RhdHVzICE9ICdzdWNjZXNzJwogICAgQU5EIHRlc3RfbmFtZSBsaWtlICd0ZXN0X3MzX3BsYWluX3Jld3JpdGFibGUlJwpPUkRFUiBCWSBjaGVja19zdGFydF90aW1lIERFU0M=

Reply:
I'll take a look in the morning. Curiously, it is happening only on the aarch64.

Reply:
Not exclusively aarch64, x86_64 is affected too, and it reproduces on my dev.
I suspect that server restart may interfere with a MOVE PARTITION that happens shortly beforehand. 
https://github.com/ClickHouse/ClickHouse/pull/78825 is the supposed fix that adds a forced merge before servers are restarted. The issue does not appear on my dev instance with the fix when the test is run repetitively.